donate.html

```diff
@@ -23,8 +23,8 @@
     <input type="hidden" name="no_note" value="1">
     <input type="hidden" name="no_shipping" value="2">
     <input type="hidden" name="rm" value="1">
-    <input type="hidden" name="return" value="https://www.accidentalshakespeare.com/thankyou">
-    <input type="hidden" name="cancel_return" value="https://www.accidentalshakespeare.com/donate">
+    <input type="hidden" name="return" value="https://www.accidentalshakespeare.org/thankyou">
+    <input type="hidden" name="cancel_return" value="https://www.accidentalshakespeare.org/donate">
```

Comment:
Can you use `{{site.url}}/thankyou` here as well? This will break. We don't have a www domain, only http://, and even that's not 100% right now.